doc_src/CreatingBackends.md

```diff
@@ -3,6 +3,10 @@
 
 See `Hammer.Backend.ETS` for a realistic example of a Hammer Backend module.
 
+
+Backends for Hammer are expected to implement the [Hammer.Backend](/hammer/Hammer.Backend.html) behaviour.
```

Comment:
Acknowledged, I think I hacked this up quickly to get it working, will create a separate issue.